Vic2ToHoI4/Data_Files/blankMod/output/common/ideas/_cultures.txt

```diff
@@ -6,16 +6,28 @@ ideas = {
 			picture = culture_aborigine
 			modifier = { }
 		}
+		culture_adelphic = {
+			removal_cost = -1
+			allowed = { always = no } allowed_civil_war = { always = yes }
+			picture = culture_adelphic
+			modifier = { }
+		}
 		culture_afar = {
 			removal_cost = -1
 			allowed = { always = no } allowed_civil_war = { always = yes }
 			picture = culture_afar
 			modifier = { }
 		}
-		culture_africaine = {
+		culture_afer = {
+			removal_cost = -1
+			allowed = { always = no } allowed_civil_war = { always = yes }
+			picture = culture_afer
+			modifier = { }
+		}
+		culture_africaine  = {
 			removal_cost = -1
 			allowed = { always = no } allowed_civil_war = { always = yes }
-			picture = culture_africaine
+			picture = culture_africaine 
```

Comment:
There's still some of these spaces. If the automated option doesn't remove them, you can use Find/Replace:
1. Set the Search Mode to extended
2. Make the Find be ```(space)\r\n```
3. Set the Replace be ```\r\n```

Just in case, do it again dropping the \r